docs/UserGuide.md

```diff
@@ -43,16 +44,16 @@ interactions recorded faster and simpler than traditional GUI applications.
 1. Double-click the file to start the app. The GUI similar to the below should appear in a few seconds. Note how the app contains some sample data.<br>
    ![Ui](images/Ui.png)
 
-1. Type the command in the command box and press Enter to execute it. e.g. typing **`help`** and pressing Enter will open the help window.<br>
+1. Type the command in the command box and press Enter to execute it. e.g. typing **`help`** and pressing Enter will show the list of commands.<br>
    Some example commands you can try:
 
    * **`list`** : Lists all clients.
 
-   * **`create`**`create 2 n/James p/97814456 e/test@test.com t/MainRemark t/owesMoney` : Creates a remark named `James` linked to the client at index 2.
+   * **`add`**`add n/Alice a/311 Clementi Ave 2 p/9191919 e/alice@gmail.com t/friends` : Adds a client named `Alice` into the address book.
 
    * **`delete`**`delete 1` : Deletes the client at the first index together with all their contacts and transactions in the current list.
 
-   * **`find`**`MacDonalds` : Finds MacDonalds in the list of clients, and displays all its details (remark, transactions).
+   * **`find`**`James` : Finds James in the list of clients, and displays all its details (remark, transactions).
```

Comment:
I think this is inaccurate. `find` has a possibility to have multiple search result, so it might not show all its details (remark, transactions).

For example, if we have 2 clients, `Alice lee`, `Alice tan`,
find Alice will filter the Client panel to have both of these names, and the transaction & remark panel will still be empty.

Reply:
Got it, will update it. Good catch!